src/Functions/translate.cpp

```diff
@@ -226,6 +268,12 @@ struct TranslateUTF8Impl
                         auto * it = map.find(*src_code_point);
                         if (it != map.end())
                         {
+                            if (it->getMapped() == max_uint32)
+                            {
+                                src += src_len;
+                                continue;
+                            }
+
```

Comment:
Here you can use only one `src += src_len;` statement for the whole if-statement:
```suggestion
                            src += src_len;
                            if (it->getMapped() == max_uint32)
                                continue;
```

and please remove the `src += src_len;` below in this if-statement